style/checkStyle.xml

```diff
@@ -74,9 +74,9 @@
     </module>
     <module name="AvoidStarImport"/>
     <module name="ImportOrder">
-      <property name="staticGroups" value="org.apache.eventmesh,org.apache,java,javax,org,com"/>
+      <property name="staticGroups" value="java,javax,org,io,net,junit,com,cn,lombok,org.apache.eventmesh"/>
       <property name="separatedStaticGroups" value="true"/>
-      <property name="groups" value="org.apache.eventmesh,org.apache,java,javax,org,com"/>
+      <property name="groups" value="java,javax,org,io,net,junit,com,lombok,org.apache.eventmesh"/>
```

Comment:
Is there any reason to change the Import order here? Refer to some project like Flink, import apache package first looks better.

Reply:
for java spec java/javax native libray import first looks better.

Reply:
Ok,I will change it and summit it again.

Reply:
I have changed it and submitted again.